### PR TITLE
DDCNL-8771 Play 3.0 upgrade

### PR DESCRIPTION
**Pull request by Nj147**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
 
   val test = Seq(
-    "uk.gov.hmrc"            %% "bootstrap-test-play-28" % bootstrapVersion % "it",
-    "com.vladsch.flexmark"   %  "flexmark-all"           % "0.62.2" % "it",
-    "com.typesafe.play"      %% "play-test"              % current  % "it",
-    "org.scalatest"          %% "scalatest"              % "3.2.12"  % "it",
-    "org.scalatestplus.play" %% "scalatestplus-play"     % "5.1.0"  % "it"
-  )
+    "uk.gov.hmrc"            %% s"bootstrap-test-$playVersion" % bootstrapVersion,
+    "com.vladsch.flexmark"   %  "flexmark-all"           % "0.62.2",
```
Comment (SandeepUmredkar) [inline project/Dependencies.scala]: We don't need flexmark all anymore.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+    "org.scalatest"          %% "scalatest"              % "3.2.12",
+    "org.scalatestplus.play" %% "scalatestplus-play"     % "5.1.0"
```
Comment (SandeepUmredkar) [inline project/Dependencies.scala]: This lib is not used anywhere and can be removed

File: project/Dependencies.scala [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
   val test = Seq(
-    "uk.gov.hmrc"            %% "bootstrap-test-play-28" % bootstrapVersion % "it",
-    "com.vladsch.flexmark"   %  "flexmark-all"           % "0.62.2" % "it",
-    "com.typesafe.play"      %% "play-test"              % current  % "it",
-    "org.scalatest"          %% "scalatest"              % "3.2.12"  % "it",
-    "org.scalatestplus.play" %% "scalatestplus-play"     % "5.1.0"  % "it"
-  )
+    "uk.gov.hmrc"            %% s"bootstrap-test-$playVersion" % bootstrapVersion,
+    "com.vladsch.flexmark"   %  "flexmark-all"           % "0.62.2",
+    "org.scalatest"          %% "scalatest"              % "3.2.12",
```
Comment (SandeepUmredkar) [inline project/Dependencies.scala]: This lib is not used anywhere in the code and can be rmeoved

File: project/Dependencies.scala (hunk 1 above)
Comment (Nj147) [reply project/Dependencies.scala]: Deleted

File: project/Dependencies.scala (hunk 2 above)
Comment (Nj147) [reply project/Dependencies.scala]: Removed

File: project/Dependencies.scala (hunk 3 above)
Comment (Nj147) [reply project/Dependencies.scala]: Done

**Review by SandeepUmredkar — APPROVED** [review]

**Pull request merged**